contract/static/src/js/contract_portal_tour.esm.js

```diff
@@ -0,0 +1,18 @@
+/** @odoo-module **/
+
+import {registry} from "@web/core/registry";
+
+registry.category("contract.tour").add("contract_portal_tour", {
```

Comment:
You're not adding the tour to the tours registry...
```suggestion
registry.category("web_tour.tours").add("contract_portal_tour", {
```

Reply:
always that simple... it is working :) thank you!